src/classes/CommandParser.ts

```diff
@@ -39,7 +40,10 @@ export = class CommandParser {
                 let value = params[key];
 
                 if (key !== 'sku') {
-                    const lowerCase = value.toLowerCase();
+                    if (typeof value !== 'string') {
+                        log.debug(`Programmer error, expect value to be of type string, got ${typeof value}`);
+                    }
+                    const lowerCase = (value as string).toLowerCase();
```

Comment:
I don't really like this

Reply:
How would you like me to resolve this?
In Hack, we would just intentionally throw an exception using `$value as string` which would propagate to `main()` and exit the process.
This is clearly not acceptable for this project, so we need some way of reporting this, but continuing the process the request.

Reply:
It shouldn't be possible for the value to not be a string, all the checks are handled in some function

Reply:
Okay, that is something I didn't know.
I'll remove the log.